ArdentHQ/dashbrd
# chore: add additional gallery logging

## Summary

https://app.clickup.com/t/8678waedw

To test:
- have a user with some NFTs (preferably not too much)
- create a gallery with said user 
- add the following lines to https://github.com/ArdentHQ/dashbrd/blob/chore/add-additional-gallery-logging/app/Jobs/FetchWalletNfts.php#L55
```
$result->nextToken = null;
$result->nfts = collect([]);
```
- run `php artisan wallets:fetch-nfts --wallet-id=<id>` with the id of the user you created galleries with
- make sure horizon is running and that you have a slack webhook set + enabled
- you should now get notifications about NFTs being removed (and the galleries will be gone)

TODO
- [x] write easy testing instructions
- [x] additional tests if needed







## Checklist
